core/server/models/webhook.js

```diff
@@ -0,0 +1,64 @@
+var ghostBookshelf = require('./base'),
+    events = require('../events'),
+    Promise = require('bluebird'),
+    Webhook,
+    Webhooks;
+
+Webhook = ghostBookshelf.Model.extend({
+    tableName: 'webhooks',
+
+    emitChange: function emitChange(event) {
+        events.emit('webhook' + '.' + event, this);
+    },
+
+    onCreated: function onCreated(model) {
+        model.emitChange('added');
+    },
+
+    onUpdated: function onUpdated(model) {
+        model.emitChange('edited');
+    },
+
+    onDestroyed: function onDestroyed(model) {
+        model.emitChange('deleted');
+    }
+}, {
+    findAllByEvent: function findAllByEvent(event, options) {
+        options = this.filterOptions(options, 'findAll');
+
+        return Webhook
+            .query({where: {event: event}})
```

Comment:
Is this correct? It's different to the `ghostBookshelf.Collection.forge` [used here](https://github.com/TryGhost/Ghost/blob/master/core/server/models/base/token.js#L28-L30), other places seem to do `ghostBookshelf.Model.findAll.call`, still others use another pattern `var postsCollection = Posts.forge(); postsCollection.query()`

Reply:
Changed this to the `var postsCollection = Posts.forge(); postsCollection.query()` pattern because I realised I was only grabbing a single record 🙈